data/binding/bindlists.go

```diff
@@ -1358,10 +1358,13 @@ type UntypedList interface {
 // ExternalUntypedList supports binding a list of interface{} values from an external variable.
 //
 // Since: 2.1
+type UntypedItemComparator func(a interface{}, b interface{}) bool
+
 type ExternalUntypedList interface {
```

Comment:
This is missing its comment. It seems to be the one that is above the `UntypedItemComparator`. That one should of course also have its own comment.

Reply:
And all these should be changed in the `gen.go`, right?

Reply:
Indeed. It was just easier to comment on the .go files as those had syntax highlighting.

Reply:
Another question: as `bindlists` is a generated file, how can I control the output to add/remove newlines? I've tried adding several newlines (for testing purposes), and they all seem to be slurped by `go generate` step so that there are now extra newlines.

Reply:
That sounds very strange. You should be able to just add and remove lines as you please. I can see the sources of the errors when I look at the generation strings.

Reply:
There should not be any strangeness - change the templates and re-generate. The newlines should be correct unless the template has been altered to introduce the line changes.

Reply:
Ah I see - you have copied usage of "{{-" which instructs the templates to remove proceeding whitespace. When the item before is a blank line you might need "{{" instead.